internal/runner/rspec.go

```diff
@@ -215,10 +215,7 @@ func (r Rspec) GetExamples(files []string) ([]plan.TestCase, error) {
 		return nil, err
 	}
 
-	cmdArgs = append(cmdArgs, "--dry-run", "--format", "json", "--out", f.Name())
-	if debug.Enabled {
-		cmdArgs = append(cmdArgs, "--format", "progress")
-	}
+	cmdArgs = append(cmdArgs, "--dry-run", "--format", "json", "--out", f.Name(), "--format", "progress")
 
 	debug.Println("Running `rspec --dry-run`")
```

Comment:
There is maybe a logic error here now, but will need to confirm.

It feels like with debug mode on, the logs will print twice?

Reply:
Since we are now appending `--format progress` every time, and log the output when the dry run fails, I don't think we should print the output in debug mode. I'll remove it.